oak/common/BUILD

```diff
@@ -71,3 +71,16 @@ cc_library(
         "@com_google_absl//absl/base",
     ],
 )
+
+cc_library(
+    name = "nonce_generator",
+    srcs = ["nonce_generator.cc"],
+    hdrs = ["nonce_generator.h"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@boringssl//:crypto",
```

Comment:
(Aside: this would be our only direct dependency on `@boringssl`, right?)

Reply:
@daviddrysdale - I don't think so. I remember seeing we pull that via asylo? we can check with bazel